greatNovels.sql

```diff
@@ -0,0 +1,149 @@
+create database greatNovels;
```

Comment:
Node projects don't ever actually have .sql files, we only used them as a temp solution until we learned migrations. This file can be deleted.